docs/cli-config.md

```diff
@@ -0,0 +1,20 @@
+---
+id: cli-config
+title: "Cli Configuration"
+---
+It is possible to tweak the behaviour of our `CliApp` specifying a custom `CliConfig` in the parameter `cliConfig` of method `CliApp.make`. By default, `CliApp.make` uses `CliConfig.default`. `CliConfig` specify how a `CliApp`
```

Comment:
This section needs to overview each setting inside `CliConfig` and discuss how it affects the Cli App, with examples for common or expected ways to configure cli apps.